### PR TITLE
Add automatic adjustment of the line width (0/1)

### PR DESCRIPTION
**Pull request by mik-laj**
Hello,

The default value that determines the number of characters used to draw a line is very high. This causes line breaks and makes it difficult to read messages. I would like to add automatic screen width detection.

I did it immediately, because it requires a small amount of work.

Thanks

### CODE REVIEW

File: oozie-to-airflow/run-sys-test [hunk 1]
```diff
@@ -412,19 +412,21 @@ function verify_arguments {
...
     echo "#    $1"
-    printf '#%.0s' {1..140}
+    printf '#%.0s' $(seq ${SEPARATOR_WIDTH})
     echo
     echo
     set -x
 }
 
 function footer {
+    SEPARATOR_WIDTH=${COLUMNS:=140}
```
Comment (potiuk) [inline oozie-to-airflow/run-sys-test]: Should be after set +x

File: oozie-to-airflow/run-sys-test [hunk 2]
```diff
@@ -412,19 +412,21 @@ function verify_arguments {
 
 function header {
     echo
-    printf '#%.0s' {1..140}
+    SEPARATOR_WIDTH=${COLUMNS:=140}
```
Comment (potiuk) [inline oozie-to-airflow/run-sys-test]: It does not work for me. I had to do "export COLUMNS" before I run the script to get the COLUMNS variable available for run-sys-tests script.

I think better solution will be to use $(tput cols). It's portable and always works.

**Comment by potiuk** [comment]
Also add reference to #188

File: oozie-to-airflow/run-sys-test (hunk 1 above)
Comment (mik-laj) [reply oozie-to-airflow/run-sys-test]: I updated the PR.

File: oozie-to-airflow/run-sys-test (hunk 2 above)
Comment (mik-laj) [reply oozie-to-airflow/run-sys-test]: Thanks for suggestion. [SOA #1](https://pl.wikipedia.org/wiki/Standardowa_odpowied%C5%BA_administratora), but I updated the PR.

**Comment by mik-laj** [comment]
@potiuk PTAL. Travis is green.

**Pull request merged**